src/riemann/codec.clj

```diff
@@ -83,7 +83,11 @@
       (when-let [v (get e k)]
         (let [a (Proto$Attribute/newBuilder)]
           (.setKey a (str (when (namespace k) (str (namespace k) \/)) (name k)))
-          (.setValue a v)
+          (try (.setValue a v)
+               (catch java.lang.ClassCastException cce
```

Comment:
Out of curiosity, any idea what the performance impact is for doing this catch on every value as opposed to round the entire encode fn?